frontend/src/components/IstioWizards/Slider/BootstrapSlider.tsx

```diff
@@ -1,63 +1,67 @@
 // Clone of Slider component to workaround issue https://github.com/patternfly/patternfly-react/issues/1221
 
-import React from 'react';
+import * as React from 'react';
 import Slider from 'bootstrap-slider-without-jquery';
-import _ from 'lodash';
+import { min } from 'lodash-es';
```

Comment:
should 'lodash-es' be imported only in one file or averywhere?

Reply:
Both `lodash` and `lodash-es` are valid, although the recommended option is to use `lodash-es` (https://www.github.com/kiali/kiali/issues/6339). Since this is a low priority maintenance task, I just update it incrementally when I touch any file that uses `lodash`.